pyrobosim/examples/test_world.py

```diff
@@ -147,4 +149,5 @@ def test_heirarcial_cleanup():
         assert len(TestWorldModeling.world.rooms) == 0
         assert len(TestWorldModeling.world.hallways) == 0
         assert len(TestWorldModeling.world.locations) == 0
-        assert len(TestWorldModeling.world.objects) == 0      
\ No newline at end of file
+        assert len(TestWorldModeling.world.objects) == 0
+     
```

Comment:
This is still complaining because there are spaces in your newline (:

Reply:
Looks like I really need that auto format lol